Old diff were not able to deal with binary files so we had to test how
quilt handles this failure. But this is not the case anymore and
modern diffs can actually deal with binary files. So drop the test for
now.

--- a/test/faildiff.test
+++ b/test/faildiff.test
@@ -22,17 +22,4 @@
 
 What happens on binary files?
 
-	$ printf "\\002\\000\\001" > test.bin
-	$ quilt add test.bin
-	> File test.bin added to patch %{P}test.diff
-
-	$ printf "\\003\\000\\001" > test.bin
-	$ quilt diff -pab --no-index
-	>~ (Files|Binary files) a/test\.bin and b/test\.bin differ
-	$ echo %{?}
-	> 1
-
-	$ quilt refresh
-	> Diff failed on file 'test.bin', aborting
-	$ echo %{?}
-	> 1
+(not tested anymore because it does not fail anymore with modern diff)
